app/lib/efile/line_data.yml

```diff
@@ -140,8 +140,6 @@ IT201_LINE_43:
   label: '43 Add lines 40, 41, and 42'
 IT201_LINE_44:
   label: '44 Subtract line 43 from line 39'
-IT201_LINE_45:
-  label: 'Arizona taxable income:  Subtract lines 43 and 44 from line 42.  If less than zero, enter “0”'
```

Comment:
this was a mistake i made on an earlier ticket, applying arizona logic to this ny form. we aren't supporting this line on the ny form so removing it.